### PR TITLE
Feature/wpmedia multiselect

### PR DESCRIPTION
**Pull request by nbradbury**
This PR addresses [this comment](https://github.com/wordpress-mobile/WordPress-Android/pull/5644#issuecomment-294508331) on #5644.

Adds multi-select to the picker used when selecting a photo from the WP media library. As with other pickers, a single tap selects a single photo and long press enables multi-select so multiple photos can be selected. When multiple items are selected, they're inserted as a `[gallery]`.

cc: @aforcier

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/media/MediaGalleryPickerActivity.java [hunk 1]
```diff
@@ -113,15 +121,12 @@ protected void onCreate(Bundle savedInstanceState) {
...
-            setTitle(R.string.select_from_media_library);
-            ActionBar actionBar = getSupportActionBar();
-            if (actionBar != null) {
-                actionBar.setDisplayHomeAsUpEnabled(true);
-            }
         } else {
             mGridAdapter.setAllowMultiselect(true);
-            mGridAdapter.setInMultiSelect(true);
-            mGridAdapter.setSelectedItems(selectedItems);
+            if (selectedItems != null && selectedItems.size() > 0) {
```
Comment (aforcier) [inline WordPress/src/main/java/org/wordpress/android/ui/media/MediaGalleryPickerActivity.java]: Small nit: we shouldn't need the `null` check here, as we set up `selectedItems` as an empty list above and haven't done anything that could `null` it (this generates a lint warning).

**Comment by aforcier** [comment]
I found an issue with the WP media library screen that predates this PR - it can be addressed in a separate PR, but we should address it before turning this on for all editors:

1. Launch the new picker
2. Select the W to open the WP media library
3. Rotate the device
4. Picker exits and returns to editor, and "An error occurred when accessing this blog" toast is displayed

**Comment by aforcier** [comment]
Looks great aside from my nit and pre-existing issue comments 👍.

It would be nice to make it more obvious how to add a gallery, but as we've discussed we can worry about that when we re-work gallery handling in general.

**Comment by nbradbury** [comment]
> I found an issue with the WP media library screen that predates this PR - it can be addressed in a separate PR

Wow, surprised I didn't see this before, especially since it's been around for a while. I would like to make that a separate PR.

**Comment by aforcier** [comment]
Had an issue ready to go for the rotation bug, opened it: https://github.com/wordpress-mobile/WordPress-Android/issues/5731

File: WordPress/src/main/java/org/wordpress/android/ui/media/MediaGalleryPickerActivity.java (hunk 1 above)
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/media/MediaGalleryPickerActivity.java]: Good catch - fixed in 903215e

**Comment by aforcier** [comment]
:shipit:!

**Pull request merged**